common/services/prismic/events.js

```diff
@@ -16,35 +16,53 @@ import {
   parseTitle,
   parseDescription,
   parseContributors,
+  parseEventFormat,
   parseImagePromo,
   parsePlace,
   asText,
+  asHtml,
   isDocumentLink,
   parseTimestamp,
   parseBoolean
 } from './parsers';
-import type {Event} from '../../model/events';
+import type {UiEvent} from '../../model/events';
 import type {PrismicDocument} from './types';
 
-function parseEventDoc(document: PrismicDocument): Event {
+// TODO: NOTE this doesn't have the A/B image test stuff in it
+function parseEventDoc(document: PrismicDocument): UiEvent {
   const data = document.data;
+
+  const interpretations = document.data.interpretations.map(interpretation => isDocumentLink(interpretation.interpretationType) ? ({
+    interpretationType: {
+      title: parseTitle(interpretation.interpretationType.data.title),
+      abbreviation: asText(interpretation.interpretationType.data.abbreviation),
+      description: asHtml(interpretation.interpretationType.data.description),
+      primaryDescription: asHtml(interpretation.interpretationType.data.primaryDescription)
+    },
+    isPrimary: Boolean(interpretation.isPrimary)
+  }) : null).filter(_ => _);
+
   return {
     id: document.id,
     title: parseTitle(data.title),
     description: asText(data.description),
     contributors: data.contributors ? parseContributors(data.contributors) : [],
     place: isDocumentLink(data.place) ? parsePlace(data.place) : null,
     promo: document.data.promo && parseImagePromo(document.data.promo),
-    audiences: [],
-    bookingEnquiryTeam: null,
-    bookingInformation: null,
-    bookingType: null,
-    cost: null,
-    format: null,
-    identifiers: [],
-    interpretations: [],
-    isDropIn: false,
-    series: [],
+    audiences: [], // TODO
+    bookingEnquiryTeam: null, // TODO
+    bookingInformation: asHtml(document.data.bookingInformation),
+    bookingType: null, // TODO
+    cost: document.data.cost,
+    format: document.data.format && parseEventFormat(document.data.format),
```

Comment:
As `parseEventFormat` isn't used anywhere else - could we not just do the parsing here, leaving the `parsers` file to just contain things that are used everywhere?